custom_components/frigate/camera.py

```diff
@@ -71,7 +72,10 @@ async def async_setup_entry(
     platform = entity_platform.async_get_current_platform()
     platform.async_register_entity_service(
         SERVICE_FAVORITE_EVENT,
-        {vol.Required(ATTR_EVENT_ID): str, vol.Required(ATTR_FAVORITE): bool},
+        {
+            vol.Required(ATTR_EVENT_ID): str,
+            vol.Optional(ATTR_FAVORITE, default=True): bool,
```

Comment:
Because you made this optional, None may now be passed to `favorite_event` and so the type hint and behavior should be updated. I assume if the `favorite` attribute is not set, it is assumed to be true.

Reply:
Also it would be good to add a test that omits this parameter and verifies the correct behavior.

Reply:
Add a test omitting the parameter.

And when passing `None`, HA bails out with the following exception:

```
MultipleInvalid: expected bool for dictionary value @ data['favorite']
```

It does not even reach the function call.